### PR TITLE
Removes support database for Trino Connection

### PR DESCRIPTION
**Pull request by TeddyCr**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by pmbrull — COMMENTED** [review]
might need to add this to the bootstrapping script as well

**Comment by TeddyCr** [comment]
> might need to add this to the bootstrapping script as well

🤔 let me do something different actually

File: ingestion/src/metadata/test_suite/api/workflow.py [hunk 1]
```diff
@@ -155,11 +155,23 @@ def _get_service_connection_from_test_case(self, table_fqn: str):
                     )
                 )
                 service_connection_config = deepcopy(service_connection.__root__.config)
-                if (
-                    hasattr(service_connection_config, "supportsDatabase")
-                    and not service_connection_config.database
-                ):
-                    service_connection_config.database = table_fqn.split(".")[1]
+                if hasattr(service_connection_config, "supportsDatabase"):
```
Comment (pmbrull) [inline ingestion/src/metadata/test_suite/api/workflow.py:158]: would it make sense to have this moved out to a utils or something? Feels like a generic "get the db from a table depending on the source" kind of thing

Comment (TeddyCr) [reply ingestion/src/metadata/test_suite/api/workflow.py:158]: Yes definitely. I am thinking of actually combining part of the profiler and test suite workflow as a lot of pieces are similar. I think we should do it separately from this though -- thinking to do the lest amount of changes for the minor release.

Comment (pmbrull) [reply ingestion/src/metadata/test_suite/api/workflow.py:158]: oh that's just for a fix pls go for it

**Review by pmbrull — APPROVED** [review]

File: ingestion/src/metadata/test_suite/api/workflow.py (hunk 1 above)
Comment (pmbrull) [reply ingestion/src/metadata/test_suite/api/workflow.py:158]: thanks mate

**Pull request merged**